core/metrics-model/src/main/java/org/hawkular/metrics/model/DataPoint.java

```diff
@@ -59,15 +58,7 @@ public DataPoint(
         tags = Collections.emptyMap();
     }
 
-    @JsonCreator(mode = Mode.PROPERTIES)
-    public DataPoint(
```

Comment:
I spent the better part of the morning getting the JSON stuff working. Jackson throws an exception if more than one constructor is annotated with ```@JsonCreator```. I went back and tested some more with the field-level annotations removed. The integration tests pass so I am going to commit the changes.